### PR TITLE
Fix unit test dependencies

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Fix unit test dependencies

Move authentication code back to controller
Lower log levels for oidc

[stage-19]

## Motivation and Context
Enables tests to run and fail correctly.

## How Has This Been Tested?
Tested changes locally. Tests run.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Reduced the oidc log level to prevent pollution of tests with console.log. Personally I don't think we need more than `WARN` as we have our own logging for oidc calls. Also moved the auth code back to the controller to prevent it from running all the time.

@ezequielc @santiagonoguez please review. Thanks!

File: gulpfile.js [hunk 1]
```diff
@@ -67,6 +67,8 @@ var unitTestFiles = [
   "web/bower_components/angular-bootstrap-colorpicker/js/bootstrap-colorpicker-module.js",
   "web/bower_components/moment/moment.js",
   "web/bower_components/angularjs-slider/dist/rzslider.min.js",
+  "web/bower_components/angularjs-slider/dist/rzslider.min.js",
```
Comment (santiagonoguez) [inline gulpfile.js:70]: is this line duplicated now ?

Comment (alex-deaconu) [reply gulpfile.js]: @santiagonoguez it is, was, fixed in the subsequent commit. Thanks.

**Review by santiagonoguez — APPROVED** [review]
looks good to me

**Review by ezequielc — APPROVED** [review]

**Pull request merged**